docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,614 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="HPCC_Terraform_deployment">
+  <title>Deploying HPCC Systems<superscript>®</superscript> with
+  Terraform</title>
+
+  <para>Manual deployments can be error-prone and inconsistent. As your
+  deployments become more customized and your need for additional resources
+  grows it can become exponentially more difficult and time consuming.</para>
+
+  <para>Fortunately, there are multiple IaC (infrastructure as code)
+  orchestration tools available that can simplify the deployment process. One
+  of those orchestration tools is Terraform. This chapter provides
+  instructions on using Terraform modules to deploy an HPCC Systems instance
+  specifically on the Azure Cloud.</para>
+
+  <para>These modules were developed by the HPCC Systems platform team for
```

Comment:
do you need the registered trademark symbol?>

Reply:
It is only required for the first and foremost instance (in any doc) not every occurrence.